Cargo.toml

```diff
@@ -17,6 +17,7 @@ members = [
     "confidential-data-hub/storage",
     "image-rs",
     "ocicrypt-rs",
+    "trustee-client",
```

Comment:
I think this name might be too general. This tool only exercises a subset of the Trustee APIs. Btw see [this proposal](https://github.com/confidential-containers/trustee/issues/529) which might end up with a similar name.

Maybe this could be called `cc-kbc-resource-getter` or `trustee-resource-getter` or `trustee-resource-client` or something else

Reply:
Thanks, I'll take a look at that proposal and think of names.